scripts/abi_artifact_generator/templates/printers.inc.jinja_template

```diff
@@ -0,0 +1,144 @@
+/*#-
```

Comment:
1. Done.
2. Would you be against postponing this to after we moved to assembly-less setup, since I'm still unsure about how much all this is going to change as the result? Maybe that will help us come up with a better name (I hate both `printer`-likes and `decoder`-likes), or 🤞, we'll even be able to assimilate this stuff into some other template.

Reply:
OK